doc/api/crypto.md

```diff
@@ -289,6 +289,25 @@ of the plaintext in bytes. See [CCM mode][].
 
 The `cipher.setAAD()` method must be called before [`cipher.update()`][].
 
+Example: Using [`cipher.setAAD()`][] method:
```

Comment:
Missing bottom reference?

Reply:
Remove the sentence entirely. It's unnecessary. The example below is obviously intended as an example of using the method that is currently being documented.